### DISCUSSION TITLE
GM Control Sheet?

### DISCUSSION THREAD

**Discussion by ethomas340** [richardwilkes/gcs#866] [General]
Richard, what is the possibility of getting a control sheet style sheet for all open characters in the window? Constraints? 

I'm sure you've answered in the past but I can't find it anywhere on Github or in the emails I have saved from the list days.

**Comment by richardwilkes** [comment]
No reason this couldn't be done. I don't know that anyone has ever requested it before, though. I'm unclear what this view would actually contain -- I suspect if I asked 10 people, I'd get 10 different answers, some of which would overlap, but many of which likely wouldn't.

Anyway, this would be a good place to have people list out specifics of what they'd like to see.

**Comment by ethomas340** [comment]
Basic Set page 568 has the official sheet. I tweaked mine in Excel, but the standard would be a great start.

**Comment by ethomas340** [comment]
And thank you for considering it!
